docs/whats-new.asciidoc

```diff
@@ -4,156 +4,149 @@
-
-*Enhancements to rule exceptions*
-
-There are several enhancements and UI improvements to rule exceptions:
-
-* When {security-guide}/add-exceptions.html[adding a rule exception], you can now create value lists for all rule types. Note that you can't use text type value lists for event correlation and threshold rules. 
-
-* If a rule has or allows endpoint exceptions, there are two exception tabs: the *Rule exceptions* tab and a new *Endpoint exceptions* tab. The *Endpoint exceptions* tab provides an easy way to manage endpoint exceptions.
-
-* You can now check how many other rules are affected by an exception by clicking *Affects _X_ rules* next to an exception list item.
+The DED integration package protects your data from being exfiltrated to external web services (technical preview only).
 
 [role="screenshot"]
-image::whats-new/images/8.5/affects-rule.png[Shows how many rules are affected by an exception]
-
-*New prebuilt rules*
+image::whats-new/images/8.6/DED-integration.png[DED integration]
 
-28 new {security-guide}/prebuilt-rules.html[prebuilt rules] were added in 8.5.0. 
+[float]
+== Entity Analytics dashboard enhancements
 
-[discrete]
-== Endpoint response enhancements 
+The following enhancements have been added to the {security-guide}/detection-entity-dashboard.html[Entity Analytics dashboard] to improve efficiency in responding to emerging threats: 
 
-*Updated messaging in response console*
+* Hover over a host name or user name to display inline actions: *Add to timeline investigation*, which adds the selected value to a new Timeline, and *Copy to Clipboard*, which copies the host name value for you to paste later.
+* Launch Timeline by clicking the number link in the *Alerts* column. 
+* Install and run a {ml} job directly from the *Notable Anomalies* table  by clicking *Run job*. 
 
-The {security-guide}/response-actions.html[response console] UI now displays a message if response action commands aren't supported by an installed version of {agent}, which must be 8.4 or later. 
+[role="screenshot"]
+image::whats-new/images/8.6/entity-dashboard.png[Entity Analytics Dashboard]
 
-*Response actions history log for all endpoints*
+[float]
+== Exceptions enhancements
 
-You can now filter and search endpoint response actions history for an endpoint. In addition, there's now a standalone {security-guide}/response-actions-history.html[Response actions history] page that shows an action log of all endpoints.
+The following enhancements have been added to the exceptions feature to improve the UI and simplify management: 
 
+* The Exceptions lists page has been renamed to *Shared Exception Lists*, where you can create, manage, and import shared exception lists.
+* There are now two types of exception lists:
+** *Default exception lists*: A default rule list is a group of exceptions that belong to a single rule. Other rules cannot use exceptions within a rule’s default rule list.
```

Comment:
Terminology is a little inconsistent -- is it a "default exception list" or a "default rule list"? I'm still not familiar enough with the new feature to know which term to use, maybe @nastasha-solomon knows?

Reply:
```suggestion
** *Default rule lists*: A default rule list is a group of exceptions that belong to a single rule. Other rules cannot use exceptions within a rule’s default rule list.
```

Reply:
Good catch, thanks @joepeeples!